content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +93,186 @@ there are any errors on the work entry, such as required fields being left blank
+appears in the :guilabel:`Company` field. It is not possible to make changes to this form.
 
 Click the :guilabel:`Create Draft Entry` button to create the payslips for the batch.
 
-Click on the :guilabel:`Payslips` button in the top right to view all the payslips for the batch.
+Click the :guilabel:`Payslips` smart button in the top right to view all the payslips for the batch.
+
+.. image:: work_entries/generate-payslips.png
+   :align: center
+   :alt: Information that appears when generating payslips.
 
 .. note::
-   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button has been
+   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button is
    clicked. After, the payslip status will change to *Done*.
 
-Payslips can be printed by clicking the box next to each payslip to print, or clicking the box next
-to :guilabel:`Reference` to select all the payslips at once. Click the :guilabel:`Print` button, and
-a PDF file will be created with all the specified payslips.
+Printing payslips
+-----------------
+
+To print payslips, first view the individual payslips by clicking the :guilabel:`Payslips` smart
+button on the batch form. Next, select the payslips to print in the payslip list. Click the box next
+to each payslip to print, or click the box next to :guilabel:`Reference` to select all the payslips
+in the list at once. Click the :guilabel:`Print` button, and a PDF file is created with all the
+specified payslips.
 
 .. image:: work_entries/print-payslips.png
    :align: center
    :alt: Print button for printing the payslips.
 
+.. note::
+   The :guilabel:`Print` button does not appear until at least one payslip is selected in the list.
+
+Time Off to Report
+==================
+
+If a time-off request is submitted for a time period that was already processed on a payslip, the
+time-off request appears in the :guilabel:`Time Off to Report`, with a status of :guilabel:`To defer
+to next payslip`. This is because the employee was already paid for that day, and it was logged as
+time spent at work, as a typical work day.
+
+In order to keep the employee's time-off balances correct, the time-off request must be applied to
+the following pay period. This not only ensures time-off request balances are current, but this also
+eliminates the need to redo work entries, cancel paychecks, and reissue paychecks.
+
+The most common scenario when this situation occurs, is when payslips are processed a day or two
+before the pay period ends, and an employee is unexpectedly sick on one of the last days of the pay
+period. The employee puts in a time-off request for a day that was already processed on a payslip as
+a regular work day. Instead of canceling the payslip, modifying the work entries, and reissuing the
+paycheck, Odoo allows for those time-off requests to be applied to the following pay period instead.
```

Comment:
```suggestion
period. The employee puts in a time off request for a day that was already processed on a payslip as
a regular work day. Instead of canceling the payslip, modifying the work entries, and reissuing the
paycheck, Odoo allows for those time off requests to be applied to the following pay period instead.
```